src/libs/protocols/protos/agent.proto

```diff
@@ -505,3 +507,55 @@ message GetMetricsRequest {}
 message Metrics {
 	string metrics = 1;
 }
+
+message VolumeStatsRequest {
+	string path = 1;
+}
+
+message VolumeStatsResponse {
+	// This field is OPTIONAL.
+	repeated VolumeUsage usage = 1;
+	// Information about the current condition of the volume.
+	// This field is OPTIONAL.
+	// This field MUST be specified if the VOLUME_CONDITION node
+	// capability is supported.
+	VolumeCondition volume_condition = 2;
+}
+message VolumeUsage {
+	enum Unit {
+		UNKNOWN = 0;
+		BYTES = 1;
+		INODES = 2;
+	}
+	// The available capacity in specified Unit. This field is OPTIONAL.
+	// The value of this field MUST NOT be negative.
+	int64 available = 1;
+
+	// The total capacity in specified Unit. This field is REQUIRED.
+	// The value of this field MUST NOT be negative.
+	int64 total = 2;
+
+	// The used capacity in specified Unit. This field is OPTIONAL.
+	// The value of this field MUST NOT be negative.
+	int64 used = 3;
+
+	// Units by which values are measured. This field is REQUIRED.
+	Unit unit = 4;
+}
+
+// VolumeCondition represents the current condition of a volume.
+message VolumeCondition {
+
+	// Normal volumes are available for use and operating optimally.
+	// An abnormal volume does not meet these criteria.
+	// This field is REQUIRED.
+	bool abnormal = 1;
+
+	// The message describing the condition of the volume.
+	// This field is REQUIRED.
+	string message = 2;
+}
+message ResizeVolumeRequest {
+	string path = 1;
```

Comment:
How about changing `path` to `device-path`, since kata-volume.go is using `device-path`.

Reply:
Updated